library/network/src/modules/DnsServerAPI.pm

```diff
@@ -654,6 +655,11 @@ sub UnquoteString {
 }
 
 sub Init {
+    if (Mode->test ()) {
```

Comment:
I must admit that I do not like this approach, although I am not sure if we have any other posibility here. If it is not possible to mock the calls to the bind rpm, I guess it is good enough.

Reply:
In the past we were just mocking the bash_output return, but now it is harder to mock the call to Yast2::Execute.

Following same approach done in https://github.com/yast/yast-users/pull/209 && https://github.com/yast/yast-squid/pull/28

Reply:
Unfortunately the old testsuite can only mock the `SCR` calls, it cannot mock the Ruby [Yast::Execute](https://github.com/yast/yast-yast2/blob/master/library/packages/src/modules/PackageSystem.rb#L350-L353). Moreover this is Perl code...

This is the only way for the old tests... :worried:

Reply:
Ok, then I am fine with the changes :) Thanks!